runtime/default/build.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/sh
```

Comment:
different then in the file above

Reply:
This one doesn't run in Lambda, it runs on the build machine.